controllers/progresses.js

```diff
@@ -107,7 +108,34 @@ const createProgress = async (req, res) => {
  */
 
 const getProgress = async (req, res) => {
+  const { dev, page = 0, size = 100, type, userId, taskId } = req.query;
   try {
+    if (dev) {
```

Comment:
this will always execute even if the value is false or any other random string

do this dev==="true"